challenges/support/challenge.md

```diff
@@ -0,0 +1,118 @@
+# Summary
+
+Investigate, troubleshoot, and remediate several issues in a simulated customer
+environment.
+
+# Rationale
+
+* It helps us understand what to expect from you as a Support Engineer, how
+  you reason about and troubleshoot production systems and how you communicate
+  with customers.
+* It helps you get a feel for what it would be like to work at Teleport, as
+  this exercise aims to simulate our day-as-usual and expose you to the type of
+  work we're doing here.
+
+We believe this technique is not only better, but also is more fun compared to
+whiteboard/quiz interviews so common in the industry. It's not without the
+downsides - it could take longer than traditional interviews.
+
+We appreciate your time and are looking forward to hacking on this project
+together.
+
+# Interview Process
+
+## Before the Interview
+
+Several days before the interview process begins, you will receive an
+invitation to a private Slack channel. This channel will contain the interview
+panel. You can ask the interview panel about the engineering culture, work-life
+balance, or anything else that you would like to learn about Teleport.
+
+## During the Interview
+
+During the 2 hours allotted for the interview, you will be given access to the
+hosts you will be debugging. Please ensure you are in the private Slack channel
+during this time, as this is where you will get the invite link.
+
+You will connect to the two faulty hosts using Teleport. You can access these
+hosts in your web browser or the CLI. If you want to use the CLI, make sure you
+[download and install](https://goteleport.com/download/) `tsh` ahead of time.
+Your cluster will be running Teleport 14.3, make sure to download the correct
+version of Teleport.
+
+You will use the three commands below during the interview process.
+
+* `tsh --proxy=interview-NAME.teleport.careers --user=interview-NAME login` to
+  login to the Teleport cluster.
+* `tsh ls` to see the list of (faulty) hosts you can connect to.
+* `tsh ssh user@interview-NAME-0` or `tsh ssh user@interview-NAME-1` to connect
```

Comment:
Is the user actually named `user`, or is this a placeholder that the candidate is expected to replace?

Reply:
It's actually user.